models/event.js

```diff
@@ -0,0 +1,83 @@
+const cozy = require('../cozyclient');
+const DOCTYPE = 'io.cozy.events';
+
+module.exports = {
+  displayName: 'Event',
+  doctype: DOCTYPE,
+  all (callback) {
+    cozy.data.defineIndex(DOCTYPE, [])
```

Comment:
Did you check it works? I think a defineIndex without attribute does not work to get all the elements. You can get an example of implementation [here](https://github.com/JbIPS/cozy-konnector-libs/blob/30b15f71b538e339f0d82fdf76c7a573f48dcf93/models/base_model.js#L9).

And better, you could directly use the baseModel and not have to reimplement everything.
You can use the baseModel like this : 

```javascript
const baseModel = require('./base_model')
module.exports = baseModel.createNew({
  name: DOCTYPE,
  // put your code here
})
```